handlers/product-move-api/src/main/scala/com/gu/productmove/zuora/Cancellation.scala

```diff
@@ -0,0 +1,47 @@
+package com.gu.productmove.zuora
+
+import com.gu.productmove.AwsS3
+import com.gu.productmove.GuStageLive.Stage
+import com.gu.productmove.zuora.{CaseId, ChargeOverrides, CreateSubscriptionResponse, ProductRatePlanChargeId, ProductRatePlanId, SubscribeToRatePlans, ZuoraAccountId}
+import com.gu.productmove.zuora.GetSubscription.GetSubscriptionResponse
+import com.gu.productmove.zuora.rest.ZuoraGet
+import sttp.capabilities.zio.ZioStreams
+import sttp.capabilities.{Effect, WebSockets}
+import sttp.client3.*
+import sttp.client3.httpclient.zio.{HttpClientZioBackend, SttpClient, send}
+import sttp.client3.ziojson.*
+import sttp.model.Uri
+import zio.json.*
+import zio.{Clock, IO, RIO, Task, UIO, URLayer, ZIO, ZLayer}
+
+import java.time.LocalDate
+
+trait Cancellation:
+  def cancel(subscriptionNumber: String, chargedThroughDate: LocalDate): ZIO[Any, String, CancellationResponse]
```

Comment:
let's change this to be in the context of the cancel call itself
```suggestion
  def cancel(subscriptionNumber: String, cancellationEffectiveDate: LocalDate): ZIO[Any, String, CancellationResponse]
```

Reply:
good idea, thanks